docs/docs/widgets/toggle-switch.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
```

Comment:
Should this be at position `2`?

Reply:
I wanted Toggle to just be below of Radio button. 
or we can serialise all the positions by how they are in the inspector? 
right now we have only 1,2,3 positions

Reply:
The list should be in alphabetical order. Can you please create another issue for that?

Reply:
yeah sure